src/apps/lwaftr/nh_fwd.lua

```diff
@@ -114,6 +120,10 @@ local function send_ipv6_cache_trigger (r, pkt, mac)
    transmit(r, ipv6_cache_trigger(pkt, mac))
 end
 
+local function random_port ()
+   return math.random(65535)
```

Comment:
That's a really unusually wide range. Take a look at https://www.cymru.com/jtk/misc/ephemeralports.html